build.psm1

```diff
@@ -499,13 +500,17 @@ Fix steps:
         $psVersion = git --git-dir="$PSSCriptRoot/.git" describe
     }
 
-    if ($Environment.IsRedHatFamily -or $Environment.IsDebian9) {
+    if ($Environment.IsRedHatFamily -or $Environment.IsDebian) {
         # add two symbolic links to system shared libraries that libmi.so is dependent on to handle
         # platform specific changes. This is the only set of platforms needed for this currently
         # as Ubuntu has these specific library files in the platform and macOS builds for itself
         # against the correct versions.
 
-        if ($Environment.IsDebian9){
+        if ($Environment.IsDebian10){
```

Comment:
We need to support Debian 11 as well since release a container image for it.

Reply:
I'll create a Deb11 image to see if the versions of the files are same or different and will update this PR

Reply:
Deb11 currently uses the same version of the libs as Deb10, so just added a check for it.